WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift

```diff
@@ -128,6 +140,40 @@ private class AccountSettingsController: SettingsController {
         }
     }
 
+    func changePassword(with service: AccountSettingsService) -> (ImmuTableRow) -> SettingsTextViewController {
+        return { row in
+            return ChangePasswordViewController() { [weak self] value in
+                DispatchQueue.main.async {
+                    SVProgressHUD.show(withStatus: Constants.changingPassword)
+                    service.updatePassword(value, finished: { (success, error) in
+                        if success {
+                            self?.refreshAccountDetails {
+                                SVProgressHUD.showSuccess(withStatus: Constants.changedPasswordSuccess)
+                            }
+                        } else {
+                            let errorMessage = error?.localizedDescription ?? Constants.changePasswordGenericError
+                            SVProgressHUD.showError(withStatus: errorMessage)
+                        }
+                        SVProgressHUD.dismiss(withDelay: TimeInterval(2))
```

Comment:
Can we increase the time the error is displayed? 2 seconds is not really long enough, especially when a long error message is displayed, like this: 

<img width="203" alt="change_password_error" src="https://user-images.githubusercontent.com/1816888/50923920-541ea100-140b-11e9-825a-7409f481f447.png">

I believe elsewhere we've started using 5 seconds, which feels much better. But it's up to you.

Reply:
Ah ok! Good to know! I'll remove it!